src/CliCommand.php

```diff
@@ -68,9 +68,10 @@ public function __invoke(array $args, array $options) {
     $databaseTableCount = $wpdb->get_col($wpdb->prepare("SELECT COUNT(*) FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = '%s'", DB_NAME))[0];
 
     // Retain user ids about to be skipped for further filters.
+    $emailsPlaceholder = implode(',', array_fill(0, count($allowedEmails), '%s'));
     $allowedUserIds = implode(',', $wpdb->get_col(
-      $wpdb->prepare("SELECT u.ID FROM {$wpdb->prefix}users u WHERE u.user_email REGEXP ('%s')", implode('|', $allowedEmails)
-    )));
+      $wpdb->prepare("SELECT u.ID FROM {$wpdb->prefix}users u WHERE u.user_email IN ({$emailsPlaceholder})", $allowedEmails)
```

Comment:
Looks good to me, but why did you use `{$wpdb->prefix}users` instead of `{$wpdb->users}? 🤔 
```suggestion
      $wpdb->prepare("SELECT u.ID FROM {$wpdb->users} u WHERE u.user_email IN ({$emailsPlaceholder})", $allowedEmails)
```

Reply:
@fabianmarz only for the sake of consistency, since tables like `woocommerce_order_items` are not available like that.